### PR TITLE
Shows a preview message when media type is not supported

### PR DESCRIPTION
**Pull request by mfrachet**
### What does it do?

In the Media Library, shows a generic error about unsupported files instead of the loading bar.

Screenshot 2021-01-29 at 16 34 54

### Why is it needed?

Right now, when uploading a file that is not supported, we show the loading bar endlessly. Without giving feedback to the user.

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/8782

### CODE REVIEW

**Review by soupette — APPROVED** [review]
LGTM  regarding the code!
We need @maevalienard's feedback regarding the displayed content when the type is supported.

**Review by HichamELBSI — DISMISSED** [review]
LGTM good job 😄

**Comment by maevalienard** [comment]
LGTM, thank you :D

File: packages/strapi-plugin-upload/admin/src/components/VideoPreview/index.js [hunk 1]
```diff
@@ -67,17 +93,35 @@ const VideoPreview = ({ hasIcon, previewUrl, src }) => {
         });
       }}
     >
-      {!snapshot && <LoadingIndicator />}
+      {!snapshot && (
+        <LoadingIndicator
+          aria-label={formatMessage(
```
Comment (soupette) [inline packages/strapi-plugin-upload/admin/src/components/VideoPreview/index.js:98]: 👍

**Review by soupette — APPROVED** [review]
LGTM!

**Pull request merged**